addons/l10n_in_ewaybill_stock_purchase/models/l10n_in_ewaybill.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class Ewaybill(models.Model):
+    _inherit = "l10n.in.ewaybill"
+
+    def _compute_document_partners_details(self):
```

Comment:
No way to avoid these 3 extra modules?  We could make it depend on sale/purchase from the start as well...  Or as we have a module l10n_in_purchase/sale_stock already, we might put some helper methods there.